api/server.ts

```
@@ -1,10 +1,24 @@
+  console.log("Successfully loaded @meeting-baas/sdk");
+} catch (error) {
+  console.error("Failed to load @meeting-baas/sdk:", error);
+}
+
 const handler = initializeMcpApiHandler(
   (server, apiKey) => {
-    // Register Meeting BaaS SDK tools with the provided API key
-    server = registerTools(server, apiKey);
+    try {
+      // Register Meeting BaaS SDK tools with the provided API key
+      server = registerTools(server, apiKey);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)